internal/npm_install/generate_build_file.ts

```diff
@@ -928,6 +964,84 @@ function findFile(pkg: Dep, m: string) {
   return undefined;
 }
 
+/**
+ * Given a pkg, return the skylark `js_library` targets for the package.
+ */
+function printPackageExperimentalDirectoryArtifacts(pkg: Dep) {
```

Comment:
hmm I'm having trouble seeing where a directory artifact originates here. is it the filegroup?

Reply:
yes. I didn't rename the targets to minimize the change set so the filegroup is still named `{pkg}__files` but it now just has a single entry in srcs which is the package directory